docs/layouts/shortcodes/guides/gatsby/headless-applicationsyaml.html

```diff
@@ -1 +1 @@
-<p>You can optionally combine the application configuration (<code>.platform.app.yaml</code>) for Gatsby and {{ .Get "name" }} into a single configuration file called <a href="/create-apps/multi-app.html#applicationsyaml"><code>applications.yaml</code></a>. Like <code>services.yaml</code> and <code>routes.yaml</code>, this file is shared across the project and resides in the <code>.platform</code> subdirectory. It does require that you explicitly define the <a href="/create-apps/multi-app.html#explicit-sourceroot">source of each application</a>, so <a href="/create-apps/multi-app.html">consult the documentation</a> for more information.</p>
+<p>You can optionally combine the application configuration (<code>.platform.app.yaml</code>) for Gatsby and {{ .Get "name" }} into a <a href="/create-apps/multi-app.html##unified-app-configuration"></a>single configuration file</a> (<code>applications.yaml</code>). Like <code>services.yaml</code> and <code>routes.yaml</code>, this file is shared across the project and resides in the <code>.platform</code> subdirectory. It does require that you explicitly define the source of each application.</p>
```

Comment:
:hammer_and_wrench:
The [link doesn't work](https://pr-2365-vbnltdi-652soceglkw4u.eu-3.platformsh.site/guides/gatsby/headless/wordpress.html#applicationsyaml), I assume that's due to the extra tag.
```suggestion
<p>You can optionally combine the application configuration (<code>.platform.app.yaml</code>) for Gatsby and {{ .Get "name" }} into a <a href="/create-apps/multi-app.html##unified-app-configuration"> single configuration file</a> (<code>applications.yaml</code>). Like <code>services.yaml</code> and <code>routes.yaml</code>, this file is shared across the project and resides in the <code>.platform</code> subdirectory. It does require that you explicitly define the source of each application.</p>
```